backend/app/controllers/teams_controller.rb

```diff
@@ -27,8 +27,13 @@ def show
   end
 
   def destroy
-    @team = Team.find(params[:id])
-    @team.destroy
+    team = Team.find(params[:id])
+    team.destroy!
+    @teams = Team.eager_load(:sports_disciplines, :target_ages).where(user_id: current_user.id)
```

Comment:
current_userからチームを引っ張って来ればwhereがいらないかと！
```suggestion
    @teams = current_user.teams.eager_load(:sports_disciplines, :target_ages)
```

Reply:
chat_rooms_controller.rbで同じような修正をしていました。

申し訳ありません。